frontend/src/types/sfcc.ts

```diff
@@ -0,0 +1,25 @@
+// subscribe:Interface
+export type SandboxSubscribe = (
+  type: string,
+  callback: (
+    payload: {
+      // config and value are user defined, can be anything
+      value: Object;
+      config: {
+        [key: string]: any;
+      };
+      isRequired: boolean;
```

Comment:
As an idea to remove any

```suggestion
export type SandboxSubscribe<Config extends Record<string, any>> = (
  type: string,
  callback: (
    payload: {
      // config and value are user defined, can be anything
      value: Object;
      config: Config;
      isRequired: boolean;
```

Reply:
This mstm, but how do I then import it into the index file?

Does something along these lines make sense?
```tsx
declare const subscribe: SandboxSubscribe<{} extends Record<string, any> ? {} : never>;
```

Reply:
Something like this would suffice when consuming it:
```tsx
declare const subscribe: SandboxSubscribe<Record<string, string | number | object>>;
```

Also bear in mind that my implementation was just a suggestion. The type definition could also just be:
```
export type SandboxSubscribe<T> = (
```

Reply:
I think I will consume it as just `declare const subscribe: SandboxSubscribe<Record<string, string | number | object>>;`.

Thanks 🤗